src/api/app/models/event/comment_event.rb

```diff
@@ -10,17 +10,17 @@ def self.included(base)
 
     def expanded_payload
       p = payload.dup
-      p['commenter'] = User.find(p['commenter'])
+      p['commenter'] = User.find_by(login: p['commenter'])
```

Comment:
Hm this might mean a window of time where we have Events with old payload. I guess we need to deal with that in a data migration.

Reply:
Good point